src/overlays/actors/ovl_En_Po_Composer/z_en_po_composer.c

```diff
@@ -100,88 +128,618 @@ static DamageTable D_80BC67E8 = {
+}
+
+void EnPoComposer_Appear(EnPoComposer* this, PlayState* play) {
+    this->actionTimer++;
+    this->actor.shape.rot.y += 0x1000;
+    if (this->actionTimer == 32) {
+        EnPoComposer_SetupIdleStill(this);
+    }
+}
+
+void EnPoComposer_SetupIdle(EnPoComposer* this) {
+    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, PO_COMPOSER_ANIM_IDLE);
+    this->actionFunc = EnPoComposer_Idle;
+}
+
+void EnPoComposer_Idle(EnPoComposer* this, PlayState* play) {
+}
+
+void EnPoComposer_SetupRaiseArms(EnPoComposer* this) {
+    this->csActionTimer = 0;
+    Actor_PlaySfx(&this->actor, NA_SE_EN_SHARP_REACTION);
+    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, PO_COMPOSER_ANIM_RAISE_ARMS);
+    this->actionFunc = EnPoComposer_RaiseArms;
+}
+
+void EnPoComposer_RaiseArms(EnPoComposer* this, PlayState* play) {
+    if ((this->csActionTimer == 0) && Animation_OnFrame(&this->skelAnime, this->skelAnime.endFrame)) {
+        this->csActionTimer++;
+        Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, PO_COMPOSER_ANIM_ARMS_RAISED);
+    }
+}
+
+void EnPoComposer_SetupLowerArms(EnPoComposer* this) {
+    this->csActionTimer = 0;
+    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, PO_COMPOSER_ANIM_LOWER_ARMS);
+    this->actionFunc = EnPoComposer_LowerArms;
+}
+
+void EnPoComposer_LowerArms(EnPoComposer* this, PlayState* play) {
+    if ((this->csActionTimer == 0) && Animation_OnFrame(&this->skelAnime, this->skelAnime.endFrame)) {
+        this->csActionTimer++;
+        Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, PO_COMPOSER_ANIM_IDLE_FACING_DOWN);
+    }
+}
+
+void EnPoComposer_SetupCutscenePlayCurse(EnPoComposer* this) {
+    this->csActionTimer = 0;
+    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, PO_COMPOSER_ANIM_BEGIN_PLAYING_CURSE);
+    this->actionFunc = EnPoComposer_CutscenePlayCurse;
+}
+
+void EnPoComposer_CutscenePlayCurse(EnPoComposer* this, PlayState* play) {
+    if ((this->csActionTimer == 0) && Animation_OnFrame(&this->skelAnime, this->skelAnime.endFrame)) {
+        this->csActionTimer++;
+        Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, PO_COMPOSER_ANIM_PLAYING_CURSE);
+    }
+}
+
+void EnPoComposer_SetupRoll(EnPoComposer* this) {
+    this->csActionTimer = 0;
+    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, PO_COMPOSER_ANIM_BEGIN_ROLLING);
+    this->actionFunc = EnPoComposer_Roll;
+}
+
+void EnPoComposer_Roll(EnPoComposer* this, PlayState* play) {
+    if (this->csActionTimer == 0) {
+        if (Animation_OnFrame(&this->skelAnime, this->skelAnime.endFrame)) {
+            this->csActionTimer++;
+            Actor_PlaySfx(&this->actor, NA_SE_EN_SHARP_REACTION);
+            Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, PO_COMPOSER_ANIM_ROLLING);
+        }
+    } else if (Animation_OnFrame(&this->skelAnime, 1.0f) && (this->lightColor.a >= 33)) {
```

Comment:
```suggestion
    } else if (Animation_OnFrame(&this->skelAnime, 1.0f) && (this->lightColor.a > 32)) {
```
Kind of nitpicky but I like the 32 instead of 33 (would also be a nice round hex number 0x20 if it wasn't a color)